### ISSUE TITLE
Reset Kubernetes window expands in size indefinitely in v1.15.0

### ISSUE THREAD

**Issue by elemental1313** [rancher-sandbox/rancher-desktop#7318]
### Actual Behavior

Going to the "Troubleshooting" menu in rancher desktop and clicking on "reset kubernetes" opens a window that attempts to increase in length indefinitely.

![image](https://github.com/user-attachments/assets/b043c94a-d839-41f7-97ef-e8417e192bb1)

### Steps to Reproduce

1. RDP into a machine you would like to use with Rancher Desktop 1.15.0 installed
2. Open Rancher Desktop
3. Wait for Rancher Desktop to be done loading
4. Click on "Troubleshooting"
5. Click on "Reset Kubernetes"
6. Experience a window that keeps increasing in height.

### Result

![image](https://github.com/user-attachments/assets/2a937d7d-6e61-47ee-90a3-2d9350409c97)

### Expected Behavior

I would expect the window to not increase in height at all.

### Rancher Desktop Version

1.15.0

### Rancher Desktop K8s Version

1.29.5

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 11 Enterprise, OS Build 22631.3880

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by VikalpRusia** [comment]
Seems this is specific for windows, on my MAC it doesn't happens

**Comment by jandubois** [comment]
@Nino-K could not reproduce on a dual-monitor Windows 10 setup.

@gunamata Can you confirm this still happens with 1.18.2?

**Comment by linghengqian** [comment]
I came from https://github.com/rancher-sandbox/rancher-desktop/issues/8488 . I can still reproduce this issue on the latest version of Rancher Desktop 1.18.2. My Windows 11 Pro 24H2 system has only one user, and the reproduction environment is a laptop with an AMD integrated graphics card and a 14-inch screen. Reproducing the bug does not require RDP and only requires a single monitor.

**Comment by jandubois** [comment]
@gunamata Since you could reproduce this, could you check if your Windows machine has an AMD graphics card? I thought you were using NVIDIA, but am not sure.

**Comment by jandubois** [comment]
So it looks like there is not much in common between the few systems that exhibit this issue.

@rak-phillip Do you have any ideas how this could be debugged by @gunamata?

**Comment by rak-phillip** [comment]
You can start by tracing the `preferred-size-changed` event[^1] for the `openDialog()` method in `pkg/rancher-desktop/window/index.ts`:

https://github.com/rancher-sandbox/rancher-desktop/blob/main/pkg/rancher-desktop/window/index.ts#L543-L549 

[^1]: https://www.electronjs.org/docs/latest/api/web-contents#event-preferred-size-changed

I suspect that there's something causing this event to emit multiple times.

**Comment by gunamata** [comment]
@rak-phillip , yes I confirm the event emits multiples times and I couldn't figure out what could be causing that. Any ideas on where to look next?
